cscore/src/main/native/include/cscore_raw.h

```diff
@@ -0,0 +1,104 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
Why is this broken out to a separate header?

Reply:
Because it's a more complicated API that is a bit easier to get into trouble with, with no OO wrapper. I could move it back in, it just seemed like because what it was it's own header would be nice.

Reply:
It seems like it should have an OO wrapper... the current API is consistent between Java and the C++ OO.

Reply:
I didn't add an oo API for java either. The frame lifetime with that becomes a bit weird